extensions/markdown-language-features/src/features/preview.ts

```diff
@@ -427,7 +427,7 @@ class MarkdownPreview extends Disposable implements WebviewResourceProvider {
 		const root = vscode.Uri.joinPath(this._resource, '../');
 		for (const src of srcs) {
 			const uri = urlToUri(src, root);
-			if (uri && uri.scheme === 'file' && !this._fileWatchersBySrc.has(src)) {
+			if (uri && !this._fileWatchersBySrc.has(src)) {
```

Comment:
Yes I'd say as a start let's ignore `http`, `https`, and `data`. We can add more if needed